src/compiler/checker.ts

```diff
@@ -14303,7 +14309,17 @@ namespace ts {
             return true;
         }
 
+        function callLikeExpressionMayHaveTypeArguments(node: CallLikeExpression): node is CallExpression | NewExpression {
+            return node.kind === SyntaxKind.CallExpression || node.kind === SyntaxKind.NewExpression;

```

Comment:
there no predicate like this utilities or wherever they live? I'm kind of surprised there's not a `isCallLikeLikeExpression` or something.